pkg/pushsync/pushsync.go

```diff
@@ -153,14 +157,18 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 // a receipt from that peer and returns error or nil based on the receiving and
 // the validity of the receipt.
 func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (*Receipt, error) {
-	r, err := ps.pushToClosest(ctx, ch)
+	stamp, err := ch.Stamp().MarshalBinary()
```

Comment:
is there a specific reason why the marshal binary is done here and not inside `pushToClosest` directly (which also could do it as it has access to `ch`)?

Reply:
No clue, I think we can move it in. Good point